perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java

```diff
@@ -1342,6 +1343,34 @@ public void removeResourceSelfServiceGroup(PerunSession sess, Resource resource,
 		return filteredGroups;
 	}
 
+	@Override
+	public void activateGroupResourceAssignment(PerunSession sess, Group group, Resource resource, boolean async) throws ResourceNotExistsException, GroupNotExistsException, PrivilegeException, WrongReferenceAttributeValueException, GroupNotDefinedOnResourceException, GroupResourceMismatchException, WrongAttributeValueException {
+		Utils.checkPerunSession(sess);
+		getResourcesManagerBl().checkResourceExists(sess, resource);
+		getPerunBl().getGroupsManagerBl().checkGroupExists(sess, group);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "activateGroupResourceAssignment_Group_Resource_boolean_policy", Arrays.asList(group, resource))) {
```

Comment:
I think we have created a new version of this method that can simplify this usage:
```suggestion
		if (!AuthzResolver.authorizedInternal(sess, "activateGroupResourceAssignment_Group_Resource_boolean_policy", group, resource)) {
```

Reply:
Done.